cdap-security/src/main/java/co/cask/cdap/security/authorization/AbstractAuthorizationService.java

```diff
@@ -210,9 +210,9 @@ private void updatePrivileges(Principal principal) throws Exception {
   private void validateCacheConfig() {
     if (!isAuthCacheEnabled()) {
       if (cacheEnabled) {
-        LOG.warn("Authorization policy caching is enabled ({} is set to true), however, this setting will have no " +
+        LOG.trace("Authorization policy caching is enabled ({} is set to true), however, this setting will have no " +
```

Comment:
can we just set isCacheEnabled to true if all three properties (security, auth, cache) are true? And then replace all isAuthCacheEnabled() to isCacheEnabled?